lib/settings_page.dart

```diff
@@ -0,0 +1,278 @@
+              ),
+              new ListTile(
+              title: Text('RED',
+                  style: TextStyle(fontWeight: FontWeight.bold, fontSize: 12)),
+              ),
+              new ListTile(
+              title: Text('YELLOW',
+                  style: TextStyle(fontWeight: FontWeight.bold, fontSize: 12)),
+              ),
+              new ListTile(
+              title: Text('GREEN',
+                  style: TextStyle(fontWeight: FontWeight.bold, fontSize: 12)),
+              ),
+              new ListTile(
+              title: Text('MAGENTA',
+                  style: TextStyle(fontWeight: FontWeight.bold, fontSize: 12)),
+              )
+            ],
+      ),
+      ExpansionTile(
+        leading: Icon(Icons.ring_volume),
+        title: Text('ALERT TYPE',
+            style: TextStyle(fontWeight: FontWeight.bold, fontSize: 15)),
+        children: <Widget>[
+              ListTile(
+              title: Text('TEXT',
+                  style: TextStyle(fontWeight: FontWeight.bold, fontSize: 12)),
+              ),
+              new ListTile(
+              title: Text('RINGER',
+                  style: TextStyle(fontWeight: FontWeight.bold, fontSize: 12)),
+              ),
+              new ListTile(
+              title: Text('EMAIL',
+                  style: TextStyle(fontWeight: FontWeight.bold, fontSize: 12)),
+              ),
+              new ListTile(
+              title: Text('NOTIFICATION',
+                  style: TextStyle(fontWeight: FontWeight.bold, fontSize: 12)),
+              ),
+              new ListTile(
+              title: Text('EVERYTHING',
+                  style: TextStyle(fontWeight: FontWeight.bold, fontSize: 12)),
+              )
+            ]
+        
+      ),
+      ExpansionTile(
+        leading: Icon(Icons.vibration),
+        title: Text('VIBRATION TYPES',
+            style: TextStyle(fontWeight: FontWeight.bold, fontSize: 15)),
+          children: <Widget>[
+              ListTile(
+              title: Text('SIREN',
+                  style: TextStyle(fontWeight: FontWeight.bold, fontSize: 12)),
+              ),
+              new ListTile(
+              title: Text('PULSE',
+                  style: TextStyle(fontWeight: FontWeight.bold, fontSize: 12)),
+              ),
+              new ListTile(
+              title: Text('TRIGGER',
+                  style: TextStyle(fontWeight: FontWeight.bold, fontSize: 12)),
+              ),
+              new ListTile(
+              title: Text('RAPID',
+                  style: TextStyle(fontWeight: FontWeight.bold, fontSize: 12)),
+              ),
+              new ListTile(
+              title: Text('HEARTBEAT',
+                  style: TextStyle(fontWeight: FontWeight.bold, fontSize: 12)),
+              )
+            ]
+      )
+    ]);
+  }
+
+  Widget _buildProfileForm() {
+    return Column(
+      mainAxisAlignment: MainAxisAlignment.center,
+      children: <Widget>[
+        _buildNameField(),
```

Comment:
these methods do the same thing, here you can just add a parameter where you pass the string field. Name it to `buildProfileField(String fieldName)`